src/e2e/puppeteer/helpers/simulateDragAndDrop.ts

```diff
@@ -0,0 +1,20 @@
+import { Page } from 'puppeteer'
+import { WindowEm } from '../../../initialize'
+
+const em = window.em as WindowEm
+
+/** Sets testFlags for simulating drag and drop process . */
+const simulateDragAndDrop = async (page: Page, drag?: boolean, drop?: boolean): Promise<void> => {
```

Comment:
Can you structure the options as an object, i.e. `{ drag: true, drop: true }`? This will make it a little more readable at the call site.